### PR TITLE
Hotfix/rabbit mq start stop implement

### PR DESCRIPTION
**Pull request by VladiTitov**
implemented Start() ands Stop() functions in RabbitMq library for manager controller

### CODE REVIEW

File: Octopus.Core.RabbitMq/Context/RabbitMqContext.cs [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
 
 namespace Octopus.Core.RabbitMq.Context
 {
     public class RabbitMqContext : IRabbitMqContext
     {
-        public IConnection Connection { get; }
+        private IConnection _connection;
+        public IConnection Connection 
+        {
+            get { return _connection ??= CreateNewRabbitConnection(); }
```
Comment (kogonidze) [inline Octopus.Core.RabbitMq/Context/RabbitMqContext.cs:13]: cool idea

File: Octopus.Core.RabbitMq/Services/RabbitMqInitializer.cs [hunk 2]
```diff
@@ -24,13 +24,11 @@ public IModel InitializeRabbitMqChannel(RabbitMqConfiguration configuration)
...
-        //    {
-        //        Channel.Close();
-        //        _connection.Close();
-        //    }
-        //}
+        public void Dispose()
+        {
+            if (!_context.Connection.IsOpen) return;
+            _context.Connection.Close();
+            _context.Connection = null;
```
[the remark this answers is not in the document]
Comment (VladiTitov) [reply Octopus.Core.RabbitMq/Services/RabbitMqInitializer.cs:31]: It's helped us to stopped service. The connection include channels, if we close the connection - this disables all subscribers and publisher services.

File: Octopus.Core.RabbitMq/Context/RabbitMqContext.cs (hunk 1 above)
Comment (VladiTitov) [reply Octopus.Core.RabbitMq/Context/RabbitMqContext.cs:13]: thank you, Vasily!

File: Octopus.Core.RabbitMq/Services/RabbitMqInitializer.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (kogonidze) [reply Octopus.Core.RabbitMq/Services/RabbitMqInitializer.cs:31]: I mean the string "`_context.Connection = null;`" Hasn`t the channel already been closed after "`_context.Connection.Close();`" ?

[the remark this answers is not in the document]
Comment (VladiTitov) [reply Octopus.Core.RabbitMq/Services/RabbitMqInitializer.cs:31]: The fact is that we close the connection, but credentials remain in it. And to start it again - you need to use `var factory = new ConnectionFactory()`. Therefore, I decided to reset the entire connection and create a new one every time

File: Octopus.Core.Loader.WebApi.Core.Application/Octopus.Core.Loader.WebApi.Core.Application.csproj [hunk 3]
```diff
@@ -1,17 +1,17 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="5.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0" />
```
Comment (Anton-Sam) [inline Octopus.Core.Loader.WebApi.Core.Application/Octopus.Core.Loader.WebApi.Core.Application.csproj:8]: cool idea

Comment (VladiTitov) [reply Octopus.Core.Loader.WebApi.Core.Application/Octopus.Core.Loader.WebApi.Core.Application.csproj:8]: Thanks!

**Review by Anton-Sam — APPROVED** [review]
Good for you

File: Octopus.Core.Loader.WebApi/Controllers/ManagerController.cs [hunk 4]
```diff
@@ -9,26 +12,34 @@ namespace Octopus.Core.Loader.WebApi.Controllers
...
         }
 
         [HttpGet]
         [Route("Start")]
         public ActionResult StartLoaderService()
         {
             _logger.LogInformation($"Starting service at: {DateTime.Now}");
-            throw new NotImplementedException();
+            _rabbitMqSubscriber.StartService(_subscribersConfiguration);
+            return Ok("Service started");
```
[the remark this answers is not in the document]
Comment (VladiTitov) [reply Octopus.Core.Loader.WebApi/Controllers/ManagerController.cs]: It's a good idea! I fixed it!

**Review by Egorkas — APPROVED** [review]

**Review by kogonidze — APPROVED** [review]
Looks good

**Pull request merged**